lib/src/Base/Geom/openturns/MeshDomain.hxx

```diff
@@ -42,7 +42,8 @@ public:
   MeshDomain();
 
   /** Default constructor */
-  explicit MeshDomain(const Mesh & mesh);
+  explicit MeshDomain(const Mesh & mesh,
+                      const Scalar barycentricCoordinatesEpsilon = ResourceMap::GetAsScalar("EnclosingSimplexAlgorithm-BarycentricCoordinatesEpsilon"));
```

Comment:
could we move non-essential parameters out of the ctor ?
I think its best to add accessors instead

Reply:
Ok done